themes/timer-hugo/assets/css/style.css

```diff
@@ -297,7 +297,8 @@ a:hover {
   background-position: center center;
   background-size: cover;
   background-attachment: fixed;
-  padding: 230px 0;
+  padding: 150px 0;
+  padding-bottom: 50px;
```

Comment:
This change doesn't render well in my setup! I'll revert for now. What were you trying to fix here?

![image](https://user-images.githubusercontent.com/2559438/81955382-4b933780-960a-11ea-9a2c-b68efc368e59.png)

Reply:
we were trying to reduce the space under the banner, JeanDi does not like it so big

Reply:
Oops, sorry, this comment didn't belong here.

Reply:
It's fixed now.